packages/E2ETest/package.json

```diff
@@ -22,10 +22,10 @@
     "deploybundleapp": "react-native run-windows --no-launch --no-packager --no-build --bundle"
   },
   "dependencies": {
-    "react": "16.8.6",
-    "react-native": "0.60.6",
-    "react-native-windows": "0.60.0-vnext.133",
-    "react-native-windows-extended": "0.60.69-11",
+    "react": "16.9.0",
+    "react-native": "0.61.5",
+    "react-native-windows": "0.61.0",
```

Comment:
>"react-native-windows": "0.61.0", [](start = 4, length = 33)

This needs to depend on beta.  Same thing for microsft-reactnative-sampleapps, and playground.

Reply:
fixed these up.

---
In reply to: [377302108](https://github.com/microsoft/react-native-windows/pull/4069#discussion_r377302108) [](ancestors = 377302108)